metadata-events/mxe-utils-avro-1.7/build.gradle

```diff
@@ -27,7 +27,21 @@ task copyOriginalMXESchemas(type: Copy) {
 }
 
 compileJava.dependsOn copyOriginalMXESchemas
+processResources.dependsOn copyOriginalMXESchemas
+
+task copyOriginalMXESchemasTest(type: Copy) {
```

Comment:
Just curious, why do you need this? I guess we don't need to copy Avro schemas to test resources? I tried removing this task & its dependencies below, build still works.

Reply:
So we needed it on our fork when we added tests to our modified consumer apps as they directly used the Avro schemas in the tests. I can remove it here if it's not needed on the OSS. I didn't think this part was needed for the test you had added which was why I removed it in the initial commit. I can remove the parts here other than the processResources dependency :)

Reply:
That sounds good. Let's remove it for now.